CMakeLists.txt

```diff
@@ -80,20 +81,33 @@ if (NOT "cxx_std_17" IN_LIST ROOT_COMPILE_FEATURES)
   message(FATAL_ERROR "You are trying to build podio against a version of ROOT that has not been built with a sufficient c++ standard. podio requires c++17")
 endif()
 #Check if Python version detected matches the version used to build ROOT
-SET(Python_FIND_FRAMEWORK LAST)
+set(Python_FIND_FRAMEWORK LAST)
```

Comment:
Please no gratuitous changes like these

Reply:
Yes, please revert for this PR. We can fix the inconsistencies in a different PR that only fixes that. Otherwise we will have commits that are mixing format / style changes and functional changes, which makes it hard to analyze the history.